docs/explanation/archive.rst

```diff
@@ -0,0 +1,348 @@
+    meaning is usually evident from the surrounding context once you are familiar
+    with the following terminologies.
+
+.. _Repositories:
+
+Repositories
+------------
+
+In the context of :term:`Package` management, *repositories* are servers
+containing sets of :term:`Packages <Package>` that a :term:`Package Manager`
+can download and install.
+
+This term refers to the *Ubuntu Package Archive* as a whole or just 
+:ref:`Suites <ArchiveSuite>`, :ref:`ArchivePockets`, or :ref:`ArchiveComponents`.
+
+.. _ArchiveSeries:
+
+Series
+------
+
+A *series* refers to the :term:`Packages <Package>` that target a specific :term:`Ubuntu`
+version. A *series* is usually referred to by its :term:`Codename`.
+
+Example *series'* are: ``mantic``, ``lunar``, ``jammy``, ``focal``, ``bionic``, ``xenial``, ``trusty``.
+
+.. note::
+
+    In practice, the terms *"Ubuntu series"* and *"Ubuntu release"* are often used
+    synonymously or are mistaken for each other. There is technically a tiny difference;
+    for example, an LTS version usually has an initial release (e.g. 22.04 LTS) and
+    multiple point releases (e.g. 22.04.1 LTS, 22.04.2 LTS), which are all part of the
+    same *series* (e.g. ``jammy``).
+
+.. _ArchivePockets:
+
+Pockets
+-------
+
+*Pockets* are :term:`Package` sub-repositories within the *Ubuntu Package Archive*.
+Every :term:`Ubuntu` :ref:`ArchiveSeries` has the following *pockets*:
+
+.. _ArchivePockets_Release:
+
+release
+~~~~~~~
+
+This *pocket* contains the :term:`Packages <Package>` that an :term:`Ubuntu`
+:ref:`ArchiveSeries` was initially released with. After the initial release of an
+:term:`Ubuntu` :ref:`ArchiveSeries`, the :term:`Packages <Package>` in this *pocket*
+are not updated (not even for security-related fixes).
+
+.. _ArchivePockets_Security:
+
+security
+~~~~~~~~
+
+This *pocket* contains security-related updates to :term:`Packages <Package>` in the
+:ref:`ArchivePockets_Release` *pocket*.
+
+.. _ArchivePockets_Updates:
+
+updates
+~~~~~~~
+
+This *pocket* contains non-security-related updates to :term:`Packages <Package>` in the
+:ref:`ArchivePockets_Release` *pocket*.
+
+.. _ArchivePockets_Proposed:
+
+proposed
+~~~~~~~~
+
+This *pocket* is a :term:`Staging Environment` the :term:`Ubuntu` community can
+opt in to verify the stability of any updates before they get deployed to a broader
+range of consumers.
+
+| After the initial release of an :term:`Ubuntu` :ref:`ArchiveSeries`, this
+  *pocket* contains non-security-related updates to :term:`Packages <Package>`
+  in the :ref:`ArchivePockets_Release` *pocket* before they get uploaded to the
+  :ref:`ArchivePockets_Updates` *pocket*.
+| Before the initial release of an :term:`Ubuntu` :ref:`ArchiveSeries`, this
+  *pocket* contains non-security-related updates to :term:`Packages <Package>`
+  in the :ref:`ArchivePockets_Release` *pocket* before they get uploaded to the
+  :ref:`ArchivePockets_Release` *pocket*.
```

Comment:
The reordering is fine. I started with after, because this is the more common situation. An LTS series has a lifespan of up to 10 years and Interim series of up to to nine months, but they are just in development for 6 months.

Reply:
The pipes actually ensures that these 2 paragraphs start on their own line without having a gap between them.

So rather than 
```
After <content>. Before <content>.
```

or

```
After <content>.

Before <content>.
```

we get 

```
After <content>.
Before <content>.
```

, but bullet points are probably okay.